.github/workflows/upload-dashboards.yaml

```diff
@@ -0,0 +1,35 @@
+name: Upload Dashboards
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
will this attempt to upload dashboards, even if there are no changes, on every push/merge to main?

Reply:
It shouldn't, because of the paths specified below, it will check to see if there have been changes made to the dashboards, and if there are, it should upload those changes. Without those changes, there shouldn't be an upload made. At least that's what I read from here --> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
I guess one other problem I see with this script is that if at least one of the files are pushed, it will trigger all the events below, meaning a revision may be the same as the previous.